### PR TITLE
Travis-ci build using docker container

### PR DESCRIPTION
**Pull request by kidus60**
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] Dymanic Accessability Plugin (DAP)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
 
 install:
-  - bundle install
-  - npm install eslint --save-dev
-  - ./node_modules/.bin/eslint src/main/content/_assets/js/*.js
+  - docker build --cache-from kidus60/travis-testing:latest -t travis-testing -f travis/Dockerfile .
+
+stages:
+  - name: Running tests
+    if: type IN (push, pull_request)
```
Comment (alohr51) [inline .travis.yml:11]: what would push entail here? I think we only want to run these on Pull Requests

Comment (kidus60) [reply .travis.yml:11]: Checking for a 'push' triggers the job to run everytime someone pushes new changes to that particular branch..pull_request only runs once when the initial pull request is set so we need both

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,26 @@
-language: ruby
-rvm:
-  - 2.6.3
+language: minimal
+
+before_install:
+  - docker pull kidus60/travis-testing:latest
```
Comment (kSee04) [inline .travis.yml:4]: Are we keeping your fork in here for these?

Comment (kidus60) [reply .travis.yml:4]: I guess for now, but I can just go ahead and creat a docker account for 'kabanero-website' push the image and update the pr

**Review by kSee04 — APPROVED** [review]

**Pull request merged**